app/controllers/admin/base_controller.rb

```diff
@@ -49,8 +49,8 @@ def preview_design_system?(next_release: false)
   end
   helper_method :preview_design_system?
 
-  def render_design_system(design_system_view, legacy_view, next_release: false)
-    if preview_design_system?(next_release:)
+  def render_design_system(design_system_view, legacy_view)
+    if self.respond_to?(:get_layout) && get_layout == "design_system"
```

Comment:
Thinking about it, you can probably do without the `respond_to?(:get_layout)` call altogether.

After all, this will only run when you call `render_design_system` in a controller. So it should never be called on a controller without that method – and it's probably a good thing if the tests explode in that scenario.

```suggestion
    if get_layout == "design_system"
```

Reply:
yup good shout. Makes sense to me